### PR TITLE
mediaconch 17.01.1

### PR DESCRIPTION
**Pull request by g-maxime**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mediaconch.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Mediaconch < Formula
   desc "Conformance checker and technical metadata reporter"
   homepage "https://mediaarea.net/MediaConch"
-  url "https://mediaarea.net/download/binary/mediaconch/17.01/MediaConch_CLI_17.01_GNU_FromSource.tar.bz2"
-  version "17.01"
-  sha256 "1fc3ae71e504db09311e01b63f46620302fcd1d83b56a818d5974397aac5bd27"
+  url "https://mediaarea.net/download/binary/mediaconch/17.01.1/MediaConch_CLI_17.01.1_GNU_FromSource.tar.bz2"
+  version "17.01.1"
+  sha256 "6decdd96e71800fe2a0c22971a57a9ac65df0267fe30e4fb497ae441657f8ead"
```
Comment (JCount) [inline Formula/mediaconch.rb]: The `sha256` seems to be incorrect. The correct one seems to be `a935084cd3fcd066de23e33a88a33cf36f447c88335f7518854495f5b8651f75`

**Review by JCount — CHANGES_REQUESTED** [review]

File: Formula/mediaconch.rb (hunk 1 above)
Comment (g-maxime) [inline Formula/mediaconch.rb]: Thanks

Comment (JCount) [inline Formula/mediaconch.rb]: No problem.

**Review by JCount — APPROVED** [review]

**Comment by JCount** [comment]
Thank you for your contribution, @g-maxime!

**Pull request closed without merging**